### PR TITLE
Docs: revised split command

### PR DESCRIPTION
**Pull request by bandantonio**
## Check yourself

- [ ] Code is linted
- [ ] Tested with redoc/reference-docs/workflows
- [ ] All new/updated code is covered with tests

## Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### CODE REVIEW

File: docs/commands/split.md [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
+openapi split [--help] [--version]
+openapi split --version
 ```
 
+## Options
 
-The command:
+Option                    | Type      | Required     | Default     | Description
+--------------------------|:---------:|:------------:|:-----------:|------------
+`entrypoint`              | `string`  | yes          | -           | Path to the API definition filename that you want to calculate statistics for
```
Comment (skadinna) [inline docs/commands/split.md:23]: issue: The description text seems to be copied from the `stats` command?

**Review by skadinna — CHANGES_REQUESTED** [review]

File: docs/commands/split.md (hunk 1 above)
Comment (bandantonio) [reply docs/commands/split.md]: Yes, sorry for that :(
Corrected

**Review by skadinna — APPROVED** [review]

File: docs/commands/split.md [hunk 2]
```diff
@@ -1,33 +1,41 @@
...
+openapi split [--help] [--version]
+openapi split --version
 ```
 
+## Options
 
-The command:
+Option                    | Type      | Required     | Default     | Description
+--------------------------|:---------:|:------------:|:-----------:|------------
+`entrypoint`              | `string`  | yes          | -           | Path to the API definition filename that you want to split apart into files
```
Comment (skadinna) [inline docs/commands/split.md:23]:
```suggestion
`entrypoint`              | `string`  | yes          | -           | Path to the API definition file that you want to split into a multi-file structure
```

**Review by skadinna — APPROVED** [review]
This looks good :+1:  
If @swapnilogale and @adamaltman are OK with this, I think we can merge it.

**Review by swapnilogale — APPROVED** [review]

**Pull request merged**